src/tictactoe_clojure/board.clj

```diff
@@ -1,28 +1,82 @@
-(ns tictactoe-clojure.board)
+(ns tictactoe-clojure.board
+  (:require [tictactoe-clojure.console :as console])
+  (:require [clojure.math.numeric-tower :as math]))
 
-(def new-board [0 1 2 3 4 5 6 7 8])
+(defn get-size
```

Comment:
`get-board-size` responsibility was to return the width of the board based on what the user wanted it to be. Describing it this way makes more sense for it to be in the game-setup namespace. I think when I was in the midst of creating the feature I was preoccupied with it being related to the board that I put it under the board-namespace. Moving forward I'll have to evaluate the underlying responsibilities of the function to determine where it should be organized.